R/cmapplot_globals.R

```diff
@@ -1,27 +1,27 @@
-# establish location for cmapplot global variables
+# Initialize environment for cmapplot global variables
 cmapplot_globals <- new.env(parent = emptyenv())
 
-# set up default font handling
-# (overridden if local machine has Whitney in .onLoad)
+# Set up default font handling
+# (Note: overridden by .onLoad() if Whitney is available)
 cmapplot_globals$use_whitney <- FALSE
 cmapplot_globals$font <- list(
   strong = list(family = "sans", face = "bold"),
   regular = list(family = "sans", face = "plain"),
   light = list(family = "sans", face = "plain"))
 
-# establish font sizes
+# Set common font sizes (bigpts)
```

Comment:
These are actually points (1/72.27 inch, I believe) - everything in the package except for fonts uses bigpts, so that fractions of inches are clean. fonts use real points, so that on import to illustrator they are read as 11 instead of 10.9, etc.

Reply:
Thanks, I'll change to pts